Description: Upstream changes introduced in version 5.22.1-4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 maxima (5.22.1-4) unstable; urgency=low
 .
   * restore changes from failed uupdate history
 .
 The person named in the Author field signed this changelog entry.
Author: Camm Maguire <camm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- maxima-5.22.1.orig/interfaces/emacs/imaxima/stamp-vti
+++ maxima-5.22.1/interfaces/emacs/imaxima/stamp-vti
@@ -1,4 +1,4 @@
-@set UPDATED 3 November 2008
+@set UPDATED 2 November 2008
 @set UPDATED-MONTH November 2008
 @set EDITION 5.22.1
 @set VERSION 5.22.1
--- maxima-5.22.1.orig/interfaces/emacs/imaxima/imaxima.info
+++ maxima-5.22.1/interfaces/emacs/imaxima/imaxima.info
@@ -1,4 +1,4 @@
-This is imaxima.info, produced by makeinfo version 4.13 from
+This is imaxima.info, produced by makeinfo version 4.7 from
 imaxima.texi.
 
 INFO-DIR-SECTION Emacs
@@ -325,9 +325,9 @@ change these options.
 
  -- User Option: imaxima-gs-options
      Options passed to `gs' when converting EPS to other image formats.
-     Older versions of Ghostscript don't support anti-aliasing. In that
-     case you might have to remove the options `-dTextAlphaBits=4' and
-     `-dGraphicsAlphaBits=4'.
+     Older versions of Ghostscript don't support anti-aliasing. In
+     that case you might have to remove the options `-dTextAlphaBits=4'
+     and `-dGraphicsAlphaBits=4'.
 
  -- User Option: imaxima-dvips-program
      Dvips executable.
@@ -474,15 +474,15 @@ File: imaxima.info,  Node: Index,  Prev:
 
 
 Tag Table:
-Node: Top531
-Node: Installation1288
-Node: Usage3332
-Node: Inline Graph5302
-Node: Options6834
-Node: Appearance7239
-Node: Line Breaking8927
-Node: Miscellaneous10280
-Node: Imath minor mode12551
-Node: Index14792
+Node: Top530
+Node: Installation1287
+Node: Usage3331
+Node: Inline Graph5301
+Node: Options6833
+Node: Appearance7238
+Node: Line Breaking8926
+Node: Miscellaneous10279
+Node: Imath minor mode12550
+Node: Index14791
 
 End Tag Table
--- maxima-5.22.1.orig/interfaces/emacs/imaxima/version.texi
+++ maxima-5.22.1/interfaces/emacs/imaxima/version.texi
@@ -1,4 +1,4 @@
-@set UPDATED 3 November 2008
+@set UPDATED 2 November 2008
 @set UPDATED-MONTH November 2008
 @set EDITION 5.22.1
 @set VERSION 5.22.1
--- maxima-5.22.1.orig/src/lisp
+++ maxima-5.22.1/src/lisp
@@ -2,4 +2,4 @@
 # -*- Shell-script -*-
 
 # Run cmucl.
-exec  "$@"
+exec  "$@"
\ No newline at end of file
